### ISSUE TITLE
It's not support mips64le

### ISSUE THREAD

**Issue by cheungxi** [evanw/esbuild#523]
Please support mips64le, Thank you very much. If will not offically support mips64le, can you provide me document to build, please?

**Comment by evanw** [comment]
Can you say more about your use case? I'm unfamiliar with the MIPS ecosystem. Some Google searches said that it's mainly used in embedded scenarios such as router hardware. Are you trying to run esbuild on a router?

Also, are you also running node on MIPS? I found a thread that says node no longer has MIPS builds. I'm trying to figure out if it makes sense to publish builds for this architecture to npm or not.

Regardless, this is how to build esbuild for `mips64le`. Clone this repo and then run this command:

```
GOOS=linux GOARCH=mips64le go build "-ldflags=-s -w" -o esbuild-linux-mips64le ./cmd/esbuild
```

**Comment by cheungxi** [comment]
> Can you say more about your use case? I'm unfamiliar with the MIPS ecosystem. Some Google searches said that it's mainly used in embedded scenarios such as router hardware. Are you trying to run esbuild on a router?
> 
> Also, are you also running node on MIPS? I found a thread that says node no longer has MIPS builds. I'm trying to figure out if it makes sense to publish builds for this architecture to npm or not.
> 
> Regardless, this is how to build esbuild for `mips64le`. Clone this repo and then run this command:
> 
> ```
> GOOS=linux GOARCH=mips64le go build "-ldflags=-s -w" -o esbuild-linux-mips64le ./cmd/esbuild
> ```

```
GOOS=linux GOARCH=mips64le go build "-ldflags=-s -w" -o esbuild-linux-mips64le ./cmd/esbuild 
go: github.com/kylelemons/godebug@v1.1.0: Get "https://proxy.golang.org/github.com/kylelemons/godebug/@v/v1.1.0.mod": dial tcp 34.64.4.17:443: i/o timeout
```
Always timeout. I want to cry :cry:

**Comment by evanw** [comment]
> Always timeout

I just removed that package since it was unnecessary. It's strange that it tried to download that package because it was only used for tests. The build still depends on `golang.org/x/sys` though so the build won't work if `https://proxy.golang.org` is blocked for you.

> On this PC, the node version is `v10.21.0`

What does `node -p process.arch` say for you on that machine?

**Comment by cheungxi** [comment]
> Always timeout 
Thank you! when I use these commands. `go build` work!
```
go env -w GO111MODULE=on
go env -w GOPROXY=https://goproxy.io,direct
```
> What does `node -p process.arch` say for you on that machine?
`mips64el`
> `GOOS=linux GOARCH=mips64le go build "-ldflags=-s -w" -o esbuild-linux-mips64le ./cmd/esbuild`
Now, It's OK.  
```
./esbuild-linux-mips64le --version
0.8.8
```
## Others
1. About `mips64el` and 'mips64le'
They all represent mips64 little endian. The name distinction is a historical issue. I think so.
2. Can mips64el be integrated into the master branch? & Upload to npm server

**Comment by evanw** [comment]
Support for `mips64le` was just released as version 0.8.9. Let me know if you encounter any problems.
